src/com2ann.py

```diff
@@ -9,19 +9,20 @@
 of additional (non-type) comments. For function definitions, we might introduce
 some formatting modifications, if the original formatting was too tricky.
 """
-import re
-import os
+from __future__ import annotations
+
+import argparse
 import ast
+import pathlib
+import re
 import sys
-import argparse
 import tokenize
-from tokenize import TokenInfo
-from enum import Enum, auto
 from collections import defaultdict
-from io import BytesIO
 from dataclasses import dataclass
-
-from typing import List, DefaultDict, Tuple, Optional, Union, Set
+from enum import Enum, auto
+from io import BytesIO
+from tokenize import TokenInfo
+from typing import Any, DefaultDict, List, Optional, Sequence, Union
```

Comment:
```suggestion
from typing import Any, DefaultDict, Sequence, Union
```

Reply:
Yes, typically, this wouldn't happen if pre-commit was configured in the repo. Even better: if pre-commit ci was configured with pycln.